### PR TITLE
Usage docs

### PR DESCRIPTION
**Pull request by todoubled**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+### Basic Usage
+
+```
+config =
+  issuerId: "issuer-id"
+  consumerKey: "consumer-key"
+  consumerSecret: "consumer-secret"
+  userId: "1"
+  key: fs.readFileSync path.join(__dirname, "app.key"), "utf8"
```
Comment (sudothinker) [inline README.md]: need a `require 'fs'` ?

Comment (britztopher) [inline README.md]: so configuring keys and such is ok, but how do you actually make calls to make requests to `getInstitutions`.

**Comment by todoubled** [comment]
Updated this with the latest based on https://github.com/hellodigit/node-intuit/pull/7.

@hellodigit/engineering ready for review

**Comment by sudothinker** [comment]
:+1:

**Pull request merged**